slurm-on-gke/image/Dockerfile

```diff
@@ -0,0 +1,121 @@
+# MIT License
+
+# Copyright (c) 2019 Giovanni Torres
```

Comment:
Is the MIT License and 2019 copy right expected here? Was it copied from another code base?

Reply:
(I see the note in the README now)

Reply:
Hey Andrew, yes, that's why some files have it, they are inspired or adapted from the original version.